vector/src/release/java/im/vector/app/core/di/FeaturesModule.kt

```diff
@@ -31,4 +33,9 @@ object FeaturesModule {
     fun providesFeatures(): VectorFeatures {
         return DefaultVectorFeatures()
     }
+
+    @Provides
+    fun providesOverrides(): VectorOverrides {
+        return DefaultVectorOverrides()
+    }
```

Comment:
Same with the above DI comment. Unfortunately I can't run builds on my machine so I can't confirm this myself but it's worth taking a look imo